packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -75,8 +75,8 @@ function BaseDrawer({
   return (
     <div
       className={classNames(
-        'fixed inset-0 z-max transition-opacity duration-300 ease-in-out',
-        !isFullScreen && 'bg-overlay-quaternary-onion',
+        'fixed inset-0 transition-opacity duration-300 ease-in-out',
+        isFullScreen ? 'z-[99]' : 'z-max bg-overlay-quaternary-onion',
```

Comment:
@sshanzel actually recently cleaned up all arbitraty zIndex because it was getting out of hand.
Cna you please use either `modal` (100) or `popup` (90) here?